docs/topics/assembly_health-check-example-application.adoc

```diff
@@ -9,20 +9,16 @@
 [id='example-health-check-{parameter-runtime-anchor}']
 = {name-example-health-check} example for {runtime}
 
-include::health-check-example-intro-paragraph.adoc[leveloffset=+1]
+Example proficiency level: 
 
-NOTE: If you are not familiar with the health check terminology, see the xref:health-check-concepts_{context}[] section first.
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
```

Comment:
See https://github.com/fabric8-launcher/launcher-documentation/pull/1308/files#r315166040

Reply:
Added my reply to your link above. I am happy to remove them if you think that is the way to go.